app/services/search/user.rb

```diff
@@ -10,8 +10,6 @@ class User
     DEFAULT_PER_PAGE = 60
     MAX_PER_PAGE = 100
 
-    # User.search_score used to take employer related fields into account, but they have since been moved to profile
```

Comment:
This comment references a method that is now gone so I think we can get rid of it too.